great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -59,7 +59,7 @@ class _FilePathDataAsset(DataAsset):
     }
 
     # General file-path DataAsset pertaining attributes.
-    batching_regex: Pattern
+    batching_regex: Pattern = MATCH_ALL_REGEX
```

Comment:
```suggestion
    batching_regex: Pattern
```
TODO: revert the schema changes this causes.
Still a good idea, but not necessary as part of the PR.